github-dark.user.css

```diff
@@ -4639,5 +4639,24 @@
   /*[[syntax-theme]]*/
   /*[[syntax-codemirror]]*/
   /*[[syntax-jupyter]]*/
-
 }
+@-moz-document domain("zuora.com") {
+  /* Styles the payment form iframe */
```

Comment:
I had to put this inside the document rule because when the generate task runs it indents it for some reason.

Reply:
Yes, this is because it also runs the [clean task](https://github.com/StylishThemes/GitHub-Dark/blob/4fb5d2a985c2e3aebd440b01447fd0e19ad8a9fc/Gruntfile.js#L424) after generate as the generator would otherwise put all selectors on a single line.